MoreLinq.Test/PartialSortTest.cs

```diff
@@ -1,6 +1,6 @@
 #region License and Terms
 // MoreLINQ - Extensions to LINQ to Objects
-// Copyright (c) 2008 Jonathan Skeet. All rights reserved.
+// Copyright (c) 2016 Felipe Sateler. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2016 Atif Aziz. All rights reserved.
```

See 13d4e52f60737a0c7658f7d57e89b724c820f0b8.